django_coverage_plugin/plugin.py

```diff
@@ -162,6 +163,15 @@ def file_tracer(self, filename):
     def file_reporter(self, filename):
         return FileReporter(filename)
 
+    def find_unexecuted_files(self, src_dir):
+        res = []
+        for i, (dirpath, dirnames, filenames) in enumerate(os.walk(src_dir)):
+            for filename in filenames:
+                if re.match(r"^[^.#~!$@%^&*()+=,]+\.html?$", filename):
```

Comment:
I like the explanation in `find_python_files` for the regex, so (and I _am_ sorry about this), you should probably put the regex back, but also include the comment from `find_python_files` as well.